can-construct.js

```diff
@@ -5,6 +5,27 @@ var dev = require("can-util/js/dev/dev");
 var makeArray = require("can-util/js/make-array/make-array");
 var types = require('can-util/js/types/types');
 var namespace = require('can-util/namespace');
+/* jshint ignore: start */
+var CanString = require('can-util/js/string/string');
+var reservedWords = [
```

Comment:
All this should be wrapped with `//steal-remove` so that it's pulled out in production.  Also, a map would be a bit faster than .indexOf.